setup.py

```diff
@@ -1,38 +1,4 @@
 """Setup module for zwave-js-server-python."""
```

Comment:
Do we still need setup.py?

Reply:
thought we did but everything seemed to work as expected without it, thanks!